services/ui-src/src/components/pages/Dashboard/MobileDashboardTable.tsx

```diff
@@ -66,10 +68,14 @@ export const MobileDashboardTable = ({
             )}
           </Text>
         </Box>
-        {reportType === "WP" && (
+        {/* Admin: Submission count */}
+        {isAdmin && (
           <Box sx={sx.labelGroup}>
+            <Text sx={sx.label}>#</Text>
             <Text sx={sx.label}>
-              {report.submissionCount === 0 ? 1 : report.submissionCount}
+              {!report.submissionCount || report.submissionCount === 0
+                ? 1
+                : report.submissionCount}{" "}
```

Comment:
line 78: Brackets with an empty string. (Probably from prettier)

Reply:
thank you for catching this! it has been removed